### ISSUE TITLE
failing LLVM assertion

### ISSUE THREAD

**Issue by JeffBezanson** [JuliaLang/julia#41686]
```
Generating REPL precompile statements... 18/31ERROR: Expected<T> must be checked before access or destruction.
Expected<T> value was in success state. (Note: Expected<T> values in success mode must still be checked prior to being destroyed).

signal (6): Aborted
in expression starting at none:0
gsignal at /lib/x86_64-linux-gnu/libc.so.6 (unknown line)
abort at /lib/x86_64-linux-gnu/libc.so.6 (unknown line)
fatalUncheckedExpected at /home/jeffbezanson/src/julia/usr/include/llvm/Support/Error.h:669
assertIsChecked at /home/jeffbezanson/src/julia/usr/include/llvm/Support/Error.h:676 [inlined]
operator-> at /home/jeffbezanson/src/julia/usr/include/llvm/Support/Error.h:566 [inlined]
find_object_file at /home/jeffbezanson/src/julia/src/debuginfo.cpp:972 [inlined]
jl_dylib_DI_for_fptr at /home/jeffbezanson/src/julia/src/debuginfo.cpp:1107
jl_getDylibFunctionInfo at /home/jeffbezanson/src/julia/src/debuginfo.cpp:1142 [inlined]
jl_getFunctionInfo at /home/jeffbezanson/src/julia/src/debuginfo.cpp:1210
jl_lookup_code_address at /home/jeffbezanson/src/julia/src/stackwalk.c:579
lookup at ./stacktraces.jl:107
stacktrace at ./stacktraces.jl:163
stacktrace at ./stacktraces.jl:161 [inlined]
scrub_repl_backtrace at ./client.jl:87
```

Cleaning everything didn't seem to fix it, but disabling LLVM assertions did.

**Comment by vtjnash** [comment]
I think you turned off support for `assert` recently (I had gone through a fixed some of these lately, but perhaps missed this one)

**Comment by JeffBezanson** [comment]
Ah ok, I guess we need to look at the llvm-config output... too bad it's in an inconvenient format...

**Comment by vtjnash** [comment]
Why not just fix the code?

**Comment by vtjnash** [comment]
fixed by 2268ee47e25503a4d8b15c1cd9044fac2f19bec3
